pdoc/__init__.py

```diff
@@ -1195,6 +1225,19 @@ def functions(self, include_inherited=True, sort=True) -> List['Function']:
             Function, include_inherited, lambda dobj: not dobj.is_method,
             sort)
 
+    def all_nested_classes(self, include_inherited=True, sort=True) -> List['Class']:
```

Comment:
With friendly `Class.classes()` available, let's make this method a private implementation detail?
```suggestion
    def _nested_classes(self, include_inherited=True, sort=True) -> List['Class']:
```

Reply:
Done.